packages/manager/src/views/WebPropertyEnvPage/components/ConfigureAccess/ConfigureAccess.tsx

```diff
@@ -0,0 +1,359 @@
+    PERMISSION_CREATION: 'PERMISSION_CREATION',
+    PERMISSION_DELETION: 'PERMISSION_DELETION',
+    ENV_CREATION: 'ENV_CREATION',
+    ENV_SYNC: 'ENV_SYNC'
+  };
+  const handleChange = (
+    checked: boolean,
+    event: React.FormEvent<HTMLInputElement>,
+    name: string,
+    email: string
+  ) => {
+    const target = event.currentTarget;
+
+    const temp = [...group.data];
+    const addAccess1 = [...addAccess];
+    const deleteAccess1 = [...deleteAccess];
+    const index = temp.findIndex((e) => e.email === email);
+    temp[index][target.name] = checked;
+    setGroup({ data: temp });
+    if (checked) {
+      addAccess1.push(target.name);
+      deleteAccess1.splice(deleteAccess1.indexOf(target.name), 1);
+    } else {
+      deleteAccess1.push(target.name);
+      addAccess1.splice(addAccess1.indexOf(target.name), 1);
+    }
+    setAddAccess(addAccess1);
+    setDeleteAccess(deleteAccess1);
+  };
+
+  const onToggleAccordian = async (id: string) => {
+    if (id === expanded) {
+      setExpanded('');
+    } else {
+      setExpanded(id);
+    }
+  };
+
+  interface Permission {
+    name: string;
+    email: string;
+    actions: string[];
+  }
+  const handleSubmit = () => {
+    const addPerm: Permission[] = [];
+    // let addflag = false;
+    const deletePerm: Permission[] = [];
+    // let deleteflag = false;
+
+    group.data.forEach((v: any) => {
+      const temp: any = {};
+      const tempActions: string[] = [];
+
+      const tempDelete: any = {};
+      const tempActionsDelete: string[] = [];
+
+      temp.name = v.name;
+      temp.email = v.email;
+      tempDelete.email = v.email;
+
+      Object.keys(v).forEach((a: string) => {
+        if (v[a] && a !== 'name' && a !== 'email' && a !== 'role') {
+          tempActions.push(a);
+        } else if (!v[a] && a !== 'name' && a !== 'email' && a !== 'role') {
+          tempActionsDelete.push(a);
+        }
+      });
+
+      // if (tempActions.length !== 0) {
+      //   addflag = true;
+      // }
+      // if (tempActionsDelete.length !== 0) {
+      //   deleteflag = true;
+      // }
+
+      temp.actions = tempActions;
+      addPerm.push(temp);
+
+      tempDelete.actions = tempActionsDelete;
+      deletePerm.push(tempDelete);
+    });
+
+    const addData = {
+      propertyIdentifier,
+      permissionDetails: addPerm
+    };
+    const deleteData = {
+      propertyIdentifier,
+      permissionDetails: deletePerm
+    };
+
+    try {
+      if (addAccess.length) {
+        useAddPermission1.mutateAsync(addData).then((res) => {
+          toast.success('Permission updated successfully');
+          return res;
+        });
+      }
+      if (deleteAccess.length) {
+        deleteMember.mutateAsync(deleteData).then((res) => {
+          toast.success('Permission updated successfully');
+          return res;
+        });
+      }
+      onClose();
+    } catch (error) {
+      // toast.error('Permission not deleted ');
```

Comment:
Please put log this issue or remove this toast
```suggestion

```

Reply:
done